### PR TITLE
Important: #95647 - Composer installations and extension usage

### PR DESCRIPTION
**Pull request by linawolf**
https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/11.4/Important-95647-ComposerInstallationsAndExtensionUsage.html
refs https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi/issues/1449

releases: master, 11.5

### CODE REVIEW

File: Documentation/4-FirstExtension/2-create-folder-structure-and-configuration-files.rst [hunk 1]
```diff
@@ -59,8 +59,11 @@ Resulting from all this, the folder structure within the extension folder
 Configuration file :file:`ext_emconf.php`
 =========================================
 
-A single configuration file named :file:`ext_emconf.php` is required by TYPO3
-to allow loading the extension. The file is located in the extension's
+The configuration file named :file:`ext_emconf.php` is required by TYPO3 in
+legacy non-Composer installations. It is recommended to keep this files in
```
Comment (mbrodala) [inline Documentation/4-FirstExtension/2-create-folder-structure-and-configuration-files.rst:63]:
```suggestion
Legacy installations. It is recommended to keep this file in
```

File: Documentation/4-FirstExtension/2-create-folder-structure-and-configuration-files.rst [hunk 2]
```diff
@@ -59,8 +59,11 @@ Resulting from all this, the folder structure within the extension folder
...
+all extensions that are being published.
+
+he file is located in the extension's
```
Comment (mbrodala) [inline Documentation/4-FirstExtension/2-create-folder-structure-and-configuration-files.rst:66]:
```suggestion
The file is located in the extension
```

File: Documentation/a-CodingGuidelines/Index.rst [hunk 3]
```diff
@@ -114,12 +114,18 @@ Every extension based on Extbase contains certain folders in the main directory:
...
+
+:file:`composer.json`
+   This file enables the extension to be installed via Composer. Read more
+   about it in the :ref:`TYPO3 Explained, composer.json <t3coreapi:composer-json>`.
+
 :file:`ext_emconf.php`
    Contains the configuration for the extension manager of TYPO3 like metadata
-   as the name, the description, and the author of the extension.
+   as the name, the description, and the author of the extension. Used in
+   legacy non-Composer based installations.
```
Comment (mbrodala) [inline Documentation/a-CodingGuidelines/Index.rst:125]:
```suggestion
   Legacy installations.
```

**Review by mbrodala — CHANGES_REQUESTED** [review]

File: Documentation/a-CodingGuidelines/Index.rst (hunk 3 above)
Comment (DavidBruchmann) [reply Documentation/a-CodingGuidelines/Index.rst:125]: I think the expression "legacy" might not be clear to everyone.

Comment (DavidBruchmann) [reply Documentation/a-CodingGuidelines/Index.rst:125]: in the top it's used "legacy non-Composer" but it's not clear that bot expressions mean actually the same

Comment (mbrodala) [reply Documentation/a-CodingGuidelines/Index.rst:125]: See the recent discussion in the docs Slack channel. This is what has been agreed upon as new name for the _Classic Mode_.

And yes, we should add links to these two at the top:

- https://docs.typo3.org/m/typo3/tutorial-getting-started/master/en-us/Installation/Install.html
- https://docs.typo3.org/m/typo3/tutorial-getting-started/master/en-us/Installation/LegacyInstallation.html

Comment (DavidBruchmann) [reply Documentation/a-CodingGuidelines/Index.rst:125]: I see. My complaint is not about the expression itself but that it's not clear for the reader perhaps.
One sentence like "Installations without composer are called legacy installations" or 
```
Installation without composer = legacy installation
```
would show it as a direct assignment.
About the location of the explanation I don't mind so much but it wouldn't be too bad having it in top of this page even if it's explained on the pages again where you posted the links. (New) users understanding a page without reading much more are the most satisfied ones I guess.

File: Documentation/4-FirstExtension/2-create-folder-structure-and-configuration-files.rst (hunk 2 above)
Comment (linawolf) [reply Documentation/4-FirstExtension/2-create-folder-structure-and-configuration-files.rst:66]: Keep reading: it is the extension's top level folder. So therefore the 's. Fixed the other one

File: Documentation/a-CodingGuidelines/Index.rst (hunk 3 above)
Comment (linawolf) [reply Documentation/a-CodingGuidelines/Index.rst:125]: On this page the word is used exactly once. I linked it to the getting started guide

**Review by tomwarwick — APPROVED** [review]

**Review by susannemoog — APPROVED** [review]

**Pull request merged**